src/ashell/shell-state.c

```diff
@@ -88,13 +88,26 @@ int32_t ashell_get_filename_buffer(const char *buf, char *destination)
 
     if (arg_len == 0) {
         *destination = '\0';
-        comms_printf(ERROR_NOT_ENOUGH_ARGUMENTS);
+        comms_println(ERROR_NOT_ENOUGH_ARGUMENTS);
         return RET_ERROR;
     }
 
     return arg_len;
 }
 
+static int8_t file_exists(const char *buf, char *filename)
```

Comment:
I foolishly saw fs_open_alloc and assumed it was a zephyr call.  Yeah makes more sense I'll take care of it there.